projects/admin/src/app/record/detail-view/patron-detail-view/patron-detail-view.component.ts

```diff
@@ -14,37 +14,41 @@
  * You should have received a copy of the GNU Affero General Public License
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
-import { Component, OnInit } from '@angular/core';
+import { Component, OnInit, OnDestroy } from '@angular/core';
 import { DetailRecord } from '@rero/ng-core/lib/record/detail/view/detail-record';
-import { Observable } from 'rxjs';
-import { PatronService } from '../../../service/patron.service';
+import { Observable, Subscription } from 'rxjs';
+import { User } from '../../../class/user';
 
 @Component({
   selector: 'admin-patron-detail-view',
   templateUrl: './patron-detail-view.component.html'
 })
-export class PatronDetailViewComponent implements OnInit, DetailRecord {
+export class PatronDetailViewComponent implements OnInit, DetailRecord, OnDestroy {
 
+  /** Data from patron we received */
   record$: Observable<any>;
 
-  record: any;
+  /** Current displayed/used patron */
+  patron: User;
 
   type: string;
 
-  constructor(private patronService: PatronService) { }
+  /** Subscription to (un)follow the record$ Observable */
+  private _subscription = new Subscription();
```

Comment:
I think you should change the name for `_subscription$` to show that's an observable.